src/lib.rs

```diff
@@ -24,3 +24,10 @@ pub mod schema;
 pub mod system;
 pub mod types;
 mod violation;
+
+/// Re-exports of ion-rust dependency that is part of our public API.
```

Comment:
The crate is `ion-rs` (which cargo rewrites to `ion_rs`) while the GitHub repo is `ion-rust`.

```suggestion
/// Re-export of the ion-rs dependency that is part of our public API.
```